backend/apps/volontulo/tests/models/test_offerstatus.py

```diff
@@ -101,6 +101,24 @@ def setUpTestData(cls):
             time_period=u'',
         )
 
+        # ongoing offer (no start date + end date)
+        Offer.objects.create(
```

Comment:
I know that we have a lot of code with `u` prefix, but we are targeting python3 so we don't need it and it's better to omit it so please remove `u` prefix at least in lines that You created (would be great to remove all occurence of `u` prefix in files that You're editing here.

For more info see: https://stackoverflow.com/a/761459